### PR TITLE
Implemented octeract stat page

### PR DESCRIPTION
**Pull request by ab3327**
Created a stat page for octeract multipliers, which shows up after the first singularity

### CODE REVIEW

File: src/Statistics.ts [hunk 1]
```diff
@@ -233,6 +234,23 @@ export const loadStatisticsCubeMultipliers = () => {
...
+
+    const arr6 = calculateOcteractMultiplier().list;
+    const map6: Record<number, { acc: number, desc: string }> = {
+        1: {acc: 2, desc: 'Ascension Score Multiplier:'},
+        2: {acc: 2, desc: 'Global Cube Multiplier:'},
+        3: {acc: 2, desc: 'Season Pass 3:'},
+        4: {acc: 2, desc: 'Season Pass Lost:'},
+        5: {acc: 2, desc: 'Cookie Upgrade 20:'},
+        6: {acc: 2, desc: 'Divine Pack:'}
+    }
```
Comment (KhafraDev) [inline src/Statistics.ts:246]: change this to an array and remove the i + 1s

File: src/Calculate.ts [hunk 2]
```diff
@@ -1236,6 +1236,30 @@ export const calculateHepteractMultiplier = (score = -1) => {
...
+export const calculateOcteractMultiplier = (score = -1) => {
+    if (score < 0) {
+        score = calculateAscensionScore().effectiveScore;
+    }
+    const corruptionLevelSum = sumContents(player.usedCorruptions.slice(2, 10))
+    const arr = [
+        // ascension score multiplier
+        (score >= 1e32) ? Math.cbrt(score / 1e32) : Math.pow(score / 1e32, 2),
+        // Global Multiplier
+        calculateAllCubeMultiplier().mult,
```
[the remark this answers is not in the document]
Comment (ab3327) [reply src/Calculate.ts:1248]: Does this look more accurate? No global multiplier for octeratcs correct?

        // ascension score multiplier

        (score >= 1e32) ? Math.cbrt(score / 1e32) : Math.pow(score / 1e32, 2),

        // season pass 3

        1 + player.shopUpgrades.seasonPass3 / 100,

        // season pass Y

        1 + player.shopUpgrades.seasonPassY / 200,

        // season pass Z

        1 + player.shopUpgrades.seasonPassZ * player.singularityCount / 100,

        // season pass lost

        1 + player.shopUpgrades.seasonPassLost / 200,

        // cube upgrade 70

        1 + +(corruptionLevelSum >= 14 * 8) * player.cubeUpgrades[70] / 10000,

        // divine pack

        1 + +(corruptionLevelSum >= 14 * 8) * +player.singularityUpgrades.divinePack.getEffect().bonus,

        // cube flame

        +player.singularityUpgrades.singCubes1.getEffect().bonus,

        // cube blaze

        +player.singularityUpgrades.singCubes2.getEffect().bonus,

        // cube inferno

        +player.singularityUpgrades.singCubes3.getEffect().bonus

        // Total Octeract Multipliers: 11
    ]`

File: src/Statistics.ts (hunk 1 above)
Comment (ab3327) [reply src/Statistics.ts:246]: I am unsure how to, I was just keeping the layout consistent with other cubes stat pages. I can look into it if necessary though.

File: src/Calculate.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Pseudonian) [reply src/Calculate.ts]: I think that's fine as is

**Pull request merged**